content/soarian/r4/foundation/other/binary.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Binary | R4 API
+---
+
+# Binary
+
+* TOC
+{:toc}
+
+## Overview
+
+The Binary resource contains the actual data content of a single consolidated clinical document architecture (C-CDA)  Summary of Episode Note (SOEN) document. The Binary resource is the target of the DocumentReference.content.attachment.url element and the content is always XML.
+
+Cerner recommends that you request a Binary resource only after obtaining the link to the resource via the reference made in DocumentReference. Cerner does not recommend starting a workflow with a Binary resource.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
+The search results include the following fields if they contain values:
+
+* [Binary id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Content type](https://hl7.org/fhir/r4/binary-definitions.html#Binary.contentType){:target="_blank"}
+* [Content](https://hl7.org/fhir/r4/binary-definitions.html#Binary.data){:target="_blank"}
+
+## Retrieve by ID
+
+List an individual Binary resource by its ID:
+
+    GET /Binary/:id
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true) %>
+
+_Implementation Note_
+
+The `Binary.read` scope and the `DocumentReference.read` scope is required.
+
+### Headers
+
+<%= headers fhir_json: true %>
```

Comment:
should these lines be included ?

DocumentReference.content.attachment.contentType should be used to set the Accept header.

This resource will not accept the application/json mime type unless the underlying binary data is json.

Reply:
Yes the underlying binary data is in json.